lib/api.js

```diff
@@ -106,9 +107,12 @@ class Api extends Emittery {
 			}
 		};
 
+		let precompiler;
+		let helpers;
+
 		try {
-			const precompiler = await this._setupPrecompiler();
-			let helpers = [];
+			precompiler = await this._setupPrecompiler(); // eslint-disable-line require-atomic-updates
```

Comment:
I think this is marked because of an issue:
https://github.com/eslint/eslint/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+require-atomic-updates